model-optimizer/extensions/middle/PreserveRuntimeInfo.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.middle.MergeNodesPermutations import MergeNodesPermutations
+from extensions.ops.transpose import Transpose
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.utils.runtime_info import OldAPIMap
+
+
+class PreserveRuntimeInfo(MiddleReplacementPattern):
```

Comment:
Can you please add a couple of unit-test? To reduce the probability of breakage if someone else will change this transformation